core/services/ardupilot_manager/mavlink_proxy/test_all.py

```diff
@@ -23,86 +24,75 @@
 serial_port_name = os.ttyname(slave_port)
 
 
-@pytest.fixture
+def endpoints_generator(role: str, offset: int = 0) -> Generator[Set[Endpoint], None, None]:
```

Comment:
Is there a reason for [7abba786e035208b051fa7edeee6bd2c60fcd036](https://github.com/bluerobotics/BlueOS/pull/2106) not being a valid alternative ?

Reply:
main reason is that that one doesn't run in parallel